src/app/pages/list/action-list/action-list.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-toolbar color="primary">
+    <div>Action List</div>
+    <div fxFlex style="margin: auto;"></div>
+    <button mat-icon-button (click)="onAddItem()"><mat-icon>add</mat-icon></button>
+</mat-toolbar>
+<mat-nav-list *ngIf="data.length > 0">
+    <pxb-info-list-item
+        [hidePadding]="true"
+        *ngFor="let i of data"
+        [class.selected]="isSelected(i)"
```

Comment:
The react example doesn't do selection behavior (just a hover state). We could pull this out to match.